lasagne/nonlinearities.py

```diff
@@ -60,6 +60,87 @@ def tanh(x):
     return theano.tensor.tanh(x)
 
 
+# scaled tanh
+class ScaledTanH(object):
```

Comment:
It's camel case for Scaled Tangens Hyperbolicus, I assume. But I think `ScaledTanh` looks more natural... maybe we should support both?